tests/ui/sse-abi-checks.rs

```diff
@@ -1,6 +1,6 @@
 //! Ensure we trigger abi_unsupported_vector_types for target features that are usually enabled
 //! on a target, but disabled in this file via a `-C` flag.
-//@ compile-flags: --crate-type=rlib --target=i686-unknown-linux-gnu -C target-feature=-sse,-sse2
+//@ compile-flags: --crate-type=rlib --target=i586-unknown-linux-gnu -C target-feature=-sse,-sse2
```

Comment:
Huh. Do we need to disable SSE and SSE2 on this target anymore?

Reply:
Ah, right, we do not... and the point here was to check that `-Ctarget-feature` disabling features is correctly checked. I guess we can't do that on x86 any more since we can't disable the relevant target features any more. Do we even still have any target that has some vector size by default which can then be disabled?

Reply:
at least Project Make Invalid States Unrepresentable seems to be coming along nicely,  even if it costs us our tests.